Makefile

```diff
@@ -4,6 +4,10 @@ DOCKER_CI_VERSION  = 2
 CONFIG_PATH       ?= $(CURDIR)/cmd/pipehub/pipehub.hcl
 WORKSPACE_PATH     = $(CURDIR)
 RAWTAG             = $(shell git tag --points-at | head -n1 | cut -c2-)
+GIT_COMMIT         = $(shell git rev-list -1 HEAD)
+GO_VERSION         = $(shell go version | cut -d' ' -f3)
+BUILT              = $(shell date)
```

Comment:
What you think of renaming this variable to `BUILT_AT`? I think it's more semantic.

Reply:
yes, I agree